.dockerignore

```diff
@@ -0,0 +1,26 @@
+# we try to aoid adding files to the docker images that change often
+# or that are not needed for running the docker image
+# tis greatly reduces the amount of times we need to rerun `npm install` when building image locally
+# https://codefresh.io/blog/not-ignore-dockerignore/
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+# consider them hidden
+.*
```

Comment:
I'm not sure if this affect the `npm install` because in other branches we are using babel and dot files are important.